src/tests.rs

```diff
@@ -40,17 +43,41 @@ mod tests {
         version: &str,
         input_cargo_toml_path: &Path,
     ) -> Result<Option<String>, Box<dyn Error>> {
-        let crates_versions = get_version_mapping_with_fallback(crate::DEFAULT_GIT_SERVER, version)
+        let mut crates_versions = get_version_mapping_with_fallback(crate::DEFAULT_GIT_SERVER, version)
             .await
             .unwrap();
 
+        let orml_crates_version = get_orml_crates_and_version(crate::DEFAULT_GIT_SERVER, &version).await.unwrap();
```

Comment:
Maybe it could return the error.

Reply:
Added that. I was sure it won't fail cause I had the files so I unwrapped it directly. But I added a `?`